docs/operator-manual/notifications/index.md

```diff
@@ -45,3 +45,19 @@ kubectl patch app <my-app> -n argocd -p '{"metadata": {"annotations": {"notifica
 ```
 
 Try syncing an application to get notified when the sync is completed.
+
+## Namespace based configuration
```

Comment:
We should probably also add a small section to the apps-in-any-namespace docs page linking back here and noting that self-service notifications are one advantage of adopting apps-in-any-namespace.